bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-a.html

```diff
@@ -0,0 +1,46 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew_38/whatsnew-base.html" %}
+
+{% add_lang_files "firefox/whatsnew_38" %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-whatsnew-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_whatsnew_38_a' %}
+{% endblock %}
+
+{% block body_id %}whatsnew-38-a{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section id="video-content">
+    <h1 class="large">{{_('From our hands to yours')}}</h1>
+    <p class="tagline">{{_('Firefox is built from the heart by thousands who believe in personal freedom on the Web.')}}</p>
+    <a id="video-thumbnail" href="{{ fx_38_video_url}}.mp4">{{_('Watch Video')}}</a>
```

Comment:
Can we use ```Watch the video```, which is the string we used so far and already available in main.lang?